sql/dql/basic-queries/where-3.md

```diff
@@ -30,38 +32,90 @@ aspects:
 ---
 ## Content
 
-### `LIKE`, `IN`
+### `LIKE`, `IN`, `BETWEEN`
 
-In a `WHERE` clause, we can also use the keywords `LIKE` and `IN`. `LIKE` allows us to define a pattern that the given attribute should conform to, such as the length or the first letter. `IN` allows us to define a list of values of which the attribute must be one. See the following examples:
+SQL provides a bunch of special operators to help you write complex querying conditions. More specifically, in this insight, we will take a look at:
+
+- the `LIKE` operator which is used to define a pattern that the given attribute should conform to
```

Comment:
I think putting a comma between "operator" and "which" for all of these would read better ([and is grammatically correct](https://www.grammarly.com/blog/comma-before-which/)).


> We'll be using:

> A [pause] which does X,
> B [pause] which does Y,
> C [pause] which does Z